packages/test/test-drivers/src/routerliciousTestDriver.ts

```diff
@@ -175,4 +175,8 @@ export class RouterliciousTestDriver implements ITestDriver {
     createCreateNewRequest(testId: string): IRequest {
         return this.createUrlResolver().createCreateNewRequest(testId);
     }
+
+    async doesDocumentExists(testId: string): Promise<boolean> {
+        throw new Error("not implemented");
```

Comment:
this will regress our stress runs. it must be implemented for all drivers if it is added, which i don't think it should be